resources/views/admin/pages/films/edit.blade.php

```diff
@@ -0,0 +1,145 @@
+@extends('admin.layout.master')
+@section('title', __('film.admin.edit.title'))
+@section('content')
+<div class="col-md-12">
+  <div class="col-md-10">
+      <div class="tile">
+        <h3 class="tile-title">@lang('film.admin.edit.title')</h3>
+        @include('admin.layout.error')
+        <div class="tile-body">
+          <form class="form-horizontal" action="{{ route('admin.films.update', $film->id) }}" method="post" enctype="multipart/form-data">
+            @csrf
+            @method('PUT')
+            <div class="form-group row">
+              <label class="control-label col-md-3">@lang('category.admin.title')</label>
+              <div class="col-md-8" >
+                <select name="categories[]" id="multiple_dropdown_select" class="form-control col-md-8" multiple>
+                  @foreach ( $categories as $category )
+                    <option value="{{ $category->id }}"
+                      @foreach ($film->cateroryFilms as $cateroryFilm)
```

Comment:
sao xứ lí phức tạp vại, thử dùng `in_array($id, $categoryIds)` em

Reply:
ở đây vì em dùng bảng `category_films` trung gian nên lấy dữ liệu theo hướng đó, em thử dùng in_array rồi nhưng chưa được ak chị

Reply:
chỗ này giờ em có thể dùng `js` hoặc `@php` để xử lí, ko nên for nhiều vại